x-pack/filebeat/input/httpjson/input_test.go

```diff
@@ -1189,6 +1188,12 @@ func TestInput(t *testing.T) {
 			conf := defaultConfig()
 			assert.NoError(t, cfg.Unpack(&conf))
 
+			var tempDir string
+			if conf.Request.Tracer != nil {
+				tempDir = t.TempDir()
+				conf.Request.Tracer.Filename = filepath.Join(tempDir, "logs", conf.Request.Tracer.Filename)
```

Comment:
```suggestion
				conf.Request.Tracer.Filename = filepath.Join(tempDir, conf.Request.Tracer.Filename)
```

This is to have the "logs" part of the path be specified entirely in the test case.